src/utils.js

```diff
@@ -24,11 +23,23 @@ export function assert(cond, msg) {
 }
 
 export function assertIsBytes(value, msg = 'value must be a Uint8Array') {
-  // XXX: Disabled until Gecko problems are resolved
-  //assert(value instanceof Uint8Array, msg);
+  // Using `value instanceof Uint8Array` seems to fail in Firefox chrome code
+  // for inscrutable reasons, so we do a less direct check.
+  assert(ArrayBuffer.isView(value), msg);
+  assert(value.BYTES_PER_ELEMENT === 1, msg);
```

Comment:
@vladikoff do either or both of these pass in Firefox chrome code, or do we need to keep looking for a suitable approach here?